packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -205,3 +205,8 @@ String getAssetBuildDirectory() {
 String getIosBuildDirectory() {
   return fs.path.join(getBuildDirectory(), 'ios');
 }
+
+/// Returns directory used by incremental compiler to store cached artifacts.
```

Comment:
Is it really 'cached artifacts' or 'cached intermediate state' ?

Reply:
Fixed the comment.